lib/Window.js

```diff
@@ -12,6 +13,7 @@ function Window(document) {
   this.document._scripting_enabled = true;
   this.document.defaultView = this;
   this.location = new Location(this, this.document._address || 'about:blank');
+  this.DOMParser = DOMParserWrapper(this);
```

Comment:
Yeah, this isn't the right way to export it.  You should instead add this to lib/impl.js, which then gets merged into Window in via `utils.expose` in the last line of this file.